test/unit_tests/view_model/bindingsSpec.js

```diff
@@ -30,17 +36,15 @@ describe('ko.bindingHandlers', () => {
     beforeEach(() => {
       element = document.createElement('div');
 
-      handler = {
-        on_enter: () => 'yay',
-      };
+      handler = {on_enter: () => 'yay'};
 
       // we need the callFake since the spyOn will overwrite the on_enter property
       spyOn(handler, 'on_enter').and.callFake(() => () => 'yay');
 
       binding.init(element, handler.on_enter);
     });
 
-    it('can execute callback when enter is pressed', () => {
+    xit('can execute callback when enter is pressed', () => {
```

Comment:
Is this exclusion on purpose?

Reply:
Yeah the problem of this one is a bit deep. 
Since we include `knockout` on the test side AND on the code side, they are 2 different knockout.
Thus the code cannot detect that an observable is an observable because it checks for the prototype which are 2 different object.

Previously we use the knockout that is on `window`. 
This will be fixed once we don't use the window.knockout at all anymore

Reply:
Edit : this one is with jQuery (same problem than explained earlier). I have another test ignored with `ko` also

Reply:
I'll add a comment